_posts/2019-05-24-announcing-lambda.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Announcing support to run OpenFaaS functions on AWS Lambda
+description: Edward Wilde introduces faas-lambda a way to run OpenFaaS functions on AWS Lambda. Learn how simple container based functions can be deployed to AWS Lambda and Kubernetes with one seamless developer experience
+date: 2019-05-24
+image: /images/openfaas-on-fargate/astronomy-atmosphere-earth-220201.jpg
+categories:
+  - fargate
+  - automation
+  - tutorial
+author_staff_member: ed
+dark_background: true
+---
+
+In this post we are going to examine some of the possible use cases for running OpenFaaS functions on AWS Lambda.
+Then we are going to take a look at some of the new components I've written to make this possible.
+Finally I would like to ask the community for some feedback on these new ideas.
+
+
+## Why would you use OpenFaaS with AWS Lambda?
+
+There could be many great reasons to run OpenFaaS functions on Lambda here are some that I've
+been considering. 
+
+**CI/CD and a unified developer experience** 
+Deploying AWS Lambda code normally requires a custom build and deployment pipeline. 
+Faas-lambda gives you the potential to eliminate this duplication and extra effort.
+
+
+**Journey to multi-cloud**
+Perhaps you started out your cloud journey on AWS and are looking to become vendor neutral and run
+on Kubernetes? Using faas-lambda you can port your existing functions to OpenFaaS and still run them on AWS Lambda
+until you are ready to migrate them over to Kubernetes
+
+**Proprietary integrations**
+There are some integration points in the AWS architecture that mandate the use of Lambdas. 
+However, there is no reason you can't create these function using OpenFaaS and it's enhanced
+developer experience.
+
+
+## How does it work?
+Hopefully I've peaked your interest? If we examine the diagram below, you can see that the
+faas-lambda provider is a standard OpenFaaS component that implements the [gateway / provider contract](link).
+The provider understands how to translate requests from the gateway into appropriate 
+responses in the AWS Lambda world. In this architecture the faas-lambda provider runs in Kubernetes
+and using the go-lang AWS SDK communicates with the AWS Lambda service APIs. Credentials to enable that communication
+are stored securely in Kubernetes as secrets, ?????or inherited transparent using IAM roles if running
```

Comment:
`?????` ?

Reply:
removed placeholder ???? linked to iam solution for EKS